src/api/definitions/country.ts

```diff
@@ -0,0 +1,7 @@
+export const CountryUrl = { get: 'country' };
```

Comment:
Countries need to be fetched form `/kyc/countries` (protected endpoint). Is a bit weird, but the allowed countries depend on the user.